style.css

```diff
@@ -630,3 +647,430 @@
+    height: 30vw;
+  }
+
+  .desktop-work-sec .desk-first-div {
+    grid-column: 1;
+    grid-row: 1;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 2vw;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-second-div {
+    grid-column: 1;
+    grid-row: 2;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 2vw;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-third-div {
+    grid-column: 1;
+    grid-row: 3;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 20px;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-fourth-div {
+    grid-column: 1;
+    grid-row: 4;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 20px;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .div-info {
+    height: 30vw;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .desktop-work-sec .div-info h2 {
+    font-size: var(--font-size-h2-desktop);
+    font-family: var(--font-family);
+    color: #091e42;
+    padding: 5px;
+    margin-top: -0.5vw;
+    margin-left: 0.8rem;
+  }
+
+  .desktop-work-sec .div-info .ul-li {
+    display: flex;
+    justify-content: space-around;
+    font-family: var(--font-family);
+    font-size: 18px;
+    color: #6b778c;
+    margin-right: 9vw;
+    margin-left: -2.6vw;
+    position: relative;
+    margin-top: -2vw;
+  }
+
+  .desktop-work-sec .div-info .sec-list {
+    font-family: var(--font-family);
+    font-size: 12px;
+    color: #6070ff;
+    position: relative;
+    margin-left: -2.1rem;
+    margin-top: -1vw;
+  }
+
+  .div-info button {
+    font-family: var(--font-family);
+    font-size: 17px;
+    color: #396df2;
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    padding: 0.5rem 1rem;
+    width: 9rem;
+    height: 3.5rem;
+    margin-left: 1rem;
+    margin-top: 0.5vw;
+    text-align: center;
+  }
+
+  .flex {
+    display: grid;
+    grid-template-columns: 30vw 50vw;
+    gap: 1rem;
+    background-color: #fff;
+    margin-top: 145rem;
+    padding: 0 6vw 6rem 6vw;
+    border-top-right-radius: 100px;
+  }
+
+  .flex .flex-box {
+    grid-column: 1;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .flex .flex-box .social-img {
+    position: relative;
+    margin-left: -3rem;
+  }
+
+  button:hover {
+    cursor: pointer;
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .contact-form form button {
+    font-family: var(--font-family);
+    font-size: 17px;
+    color: #396df2;
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
```

Comment:
- [ ] Please 🙏 adjust the padding of your `get in touch` button in the contact form mobile version to make it look like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0)

_**Figma**_

<img width="110" alt="Screenshot 2023-02-23 at 10 48 43 PM" src="https://user-images.githubusercontent.com/44798044/220988602-cbbce68f-d5bd-4b13-9ba5-299ccef0790f.png">

**_Yours_**

<img width="177" alt="Screenshot 2023-02-23 at 9 28 10 PM" src="https://user-images.githubusercontent.com/44798044/220988438-0b04937d-f223-4495-8d10-9e751100bb45.png">